### ISSUE TITLE
ROS-indigo Vaccum gripper Plugin support

### ISSUE THREAD

**Issue by ghost** [ros-simulation/gazebo_ros_pkgs#576]
Hi  @wkentaro
I am using your vaccum gripper plugin for UR5-Robot.
Can it will work with ROS-indigo and Gazebo Ver 2 with Ubuntu 14.04 LTS?

**Comment by wkentaro** [comment]
Yes, it should be. I originally created the plugin in the same environment as yours.
Maybe https://github.com/ros-simulation/gazebo_ros_pkgs/pull/482 is needed to make the plugin works stably with different robot.
I originally used with Baxter robot, and then found #482 is needed for other robots (ex. fanuc for me).
